internal/keys/keys.go

```diff
@@ -0,0 +1,158 @@
+package keys
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	cryptorand "crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"errors"
+	"math"
+	"math/big"
+	"time"
+
+	certutil "k8s.io/client-go/util/cert"
+	"k8s.io/client-go/util/keyutil"
+)
+
+const duration365d = time.Hour * 24 * 365
```

Comment:
nit: isn't 365d a year? 🤔

Reply:
Not exactly. https://pumas.nasa.gov/examples/index.php?id=46 <- a year is actually more like 365.2422 days

Reply:
What a troll :trollface: !